LevelUp.Pos.ProposedOrders/ProposedOrderCalculator.cs

```diff
@@ -148,29 +186,5 @@ internal static int CalculateAdjustedCustomerPaymentAmount(int totalOutstandingA
 
             return Math.Max(0, adjustedExemptionAmount);
         }
-
```

Comment:
This is the removal I referred to.  This was internal and superfluous.  There will be no impact to dependent systems.